spec/features/4_mylikes_spec.rb

```diff
@@ -14,23 +11,32 @@
     expect(page.status_code).to be(200) # 200 is "OK"
   end
 
-  scenario "only shows photos current user liked", points: 2 do
-    user_1 = FactoryGirl.create(:user, :username => "1", :email => "1@m.com")
-    user_2 = FactoryGirl.create(:user, :username => "2", :email => "2@m.com")
-    photo_1 = FactoryGirl.create(:photo, :user_id => user_1.id)
-    photo_2 = FactoryGirl.create(:photo, :user_id => user_1.id)
-    photo_3 = FactoryGirl.create(:photo, :user_id => user_2.id)
-    like_1 = FactoryGirl.create(:like, :user_id => user_2.id, :photo_id => photo_1.id)
-    like_2 = FactoryGirl.create(:like, :user_id => user_2.id, :photo_id => photo_3.id)
-    login_as(user_2, :scope => :user)
-
-    visit "/my_likes"
-
-    expect(page.status_code).to be(200) # 200 is "OK"
-    expect(page).to have_content(like_1.photo.caption)
-    expect(page).to have_css("img[src*='#{like_1.photo.image}']")
-    expect(page).to have_content(like_2.photo.caption)
-    expect(page).to have_css("img[src*='#{like_2.photo.image}']")
+  context "my likes" do
+    before(:each) do
+      @user_1 = FactoryGirl.create(:user, :username => "1", :email => "1@m.com")
+      @user_2 = FactoryGirl.create(:user, :username => "2", :email => "2@m.com")
+      @photo_1 = FactoryGirl.create(:photo, :user_id => @user_1.id)
+      @photo_2 = FactoryGirl.create(:photo, :user_id => @user_1.id)
+      @photo_3 = FactoryGirl.create(:photo, :user_id => @user_2.id)
+      @like_1 = FactoryGirl.create(:like, :user_id => @user_2.id, :photo_id => @photo_1.id)
+      @like_2 = FactoryGirl.create(:like, :user_id => @user_2.id, :photo_id => @photo_3.id)
+      login_as(@user_2, :scope => :user)
+      visit "/my_likes"
+    end
+
+    scenario "shows photos current user liked", points: 1 do
+      @user_2.likes.each do |like|
+        expect(page).to have_content(like.photo.caption)
+        expect(page).to have_css("img[src*='#{like.photo.image}']")
+      end
+    end
+
+    scenario "not shows photos current user liked", points: 1 do
```

Comment:
These two scenarios seem clunky. Maybe the two tests should be `shows photos a user has liked` and `does not show photos a user hasn't liked`?